kafka-commons/src/main/java/uk/gov/homeoffice/digital/sas/kafka/producer/KafkaProducerService.java

```diff
@@ -27,8 +37,43 @@ public KafkaProducerService(
 
   @SuppressWarnings("unchecked")
```

Comment:
If you action suggestion below, I believe this `SuppressWarnings` annotation can then be removed

Reply:
Your thoughts are correct. I have actioned the below and removed the annotation